android/templates/snapshot/src/main/java/com/badoo/ribs/template/rib_with_view/foo_bar/FooBarView.kt

```diff
@@ -37,11 +39,20 @@ class FooBarViewImpl private constructor(
     ) : FooBarView.Factory {
         override fun invoke(deps: Nothing?): (ViewGroup) -> FooBarView = {
             FooBarViewImpl(
-                com.badoo.ribs.customisation.inflate(it, layoutRes)
+                inflate(it, layoutRes)
             )
         }
     }
 
-    override fun accept(vm: FooBarView.ViewModel) {
+    // feel free to check https://badoo.github.io/MVICore/extras/modelwatcher/ for usage examples
```

Comment:
do we need this comment? maybe we can put it in `modelWatcher` javadoc? currently, we should always remove it